src/routes/create_role.ts

```diff
@@ -0,0 +1,41 @@
+import express from "express"
+import { Client } from "../entities/Client";
+import { Role } from "../entities/Role";
+import { RoleTypes } from "../entities/Role";
+const router = express.Router();
+
+router.post("/api/client/:user_id/role", async(req,res)=>{
+    const user_id = parseInt(req.params.user_id, 10);
+    const {role} = req.body
```

Comment:
what if the req.body does not have role ? the api will crash

Reply:
Will check this out and edit it.